src/components/accessories/infoBox/InfoBox.tsx

```diff
@@ -2,11 +2,23 @@ import classNames from "classnames";
 import React, { FunctionComponent } from "react";
 import { IProps } from "./types";
 import "./styles.scss";
+import InfoIcon from "@material-ui/icons/Info";
+import ErrorIcon from "@material-ui/icons/Error";
+import WarningIcon from "@material-ui/icons/Warning";
 
 const InfoBox: FunctionComponent<IProps> = ({ type, message }) => {
   return (
     <div className={classNames(`infoBox ${type}`)}>
-      <h5>{message}</h5>
+      <h5>
+        {type === "info" ? (
+          <InfoIcon fontSize="small" />
+        ) : type === "warning" ? (
+          <WarningIcon fontSize="small" />
+        ) : (
+          <ErrorIcon fontSize="small" />
+        )}
+        &nbsp; {message}
+      </h5>
```

Comment:
I would prefer use a separate method in order to render the icon and a switch case in order to catch the conditions. Plus, if the type props has a finite status I would use a enum.

Reply:
The space before the message I would add it on the icon itselft with the css (e.g margin-right: 10px).

Reply:
Ok thanks for the suggestions. Could you check again please?